django/contrib/auth/hashers.py

```diff
@@ -515,3 +515,72 @@ def safe_summary(self, encoded):
             (_('salt'), salt),
             (_('hash'), mask_hash(data, show=3)),
         ])
+
+
+class Argon2PasswordHasher(BasePasswordHasher):
```

Comment:
The hashers seem to be ordered based on where they appear in settings so I'd move it up.